src/dj_notebook/config_helper.py

```diff
@@ -0,0 +1,124 @@
+import ast
+import os
+from pathlib import Path
+from typing import Generator, Tuple
+
+from dotenv import load_dotenv
+
+
+# taken from dotenv, which declares a similar type (but it doesn't look public...)
+# review note: the | syntax is new in python 3.10. If older pythons are generally being supported here, this should be
```

Comment:
Older Pythons are not something we should support in this stage of the project. Especially as it is in the "cook up the API" mode of early stage projects. In fact, I've toyed with removed Python 3.9 support - this might just push us over to removing it.